### DISCUSSION TITLE
Quirks description in help message

### DISCUSSION THREAD

**Discussion by jpc4242** [tliron/go-puccini#55] [Ideas]
Would it make sense to have a way to show all available quirks and their meaning from the command line?
Thsi could be a dedicated command (`puccini-tosca quirks`) or juts show the info as part of the output for the `version` command

**Comment by tliron** [comment]
Very good suggestion! I think it could be in incorporated into the `--help`? It would be a big help screen, but many apps have that.

**Comment by jpc4242** [comment]
Yes, `--help` sounds the natural place for this.
Should the Quirk datatype change into something that may contain the name and the description so the help is always in sync with the code? Or are you thinking about just writing a static help text?

**Comment by tliron** [comment]
I'm always in favor of not repeating work :) Perhaps the "embed" script can be extended to pull the description in and update it in all the places.
